eclipse-collections-code-generator/src/main/resources/api/bag/immutablePrimitiveBag.stg

```diff
@@ -14,11 +14,16 @@ body(type, name) ::= <<
 
 package org.eclipse.collections.api.bag.primitive;
 
+import java.util.Collection;
+import java.util.Set;
```

Comment:
I was thinking about the primitive `Set`, not `java.util.Set` (primitive sets don't implement that).